app/brave_generated_resources.grd

```diff
@@ -964,6 +964,18 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         <message name="IDS_TAB_CXMENU_CLOSE_DUPLICATE_TABS" desc="The label of the 'Close duplicate tabs' tab context menu item">
           Close duplicate tabs
         </message>
+        <message name="IDS_IDC_NEW_SPLIT_VIEW" desc="The label of the 'New Split View' tab context menu item">
+          New split view
+        </message>
+        <message name="IDS_IDC_CLOSE_SPLIT_VIEW" desc="The label of the 'Close Split View' tab context menu item">
+          Close split view
+        </message>
+        <message name="IDS_TAB_CXMENU_TILE_TABS" desc="The label of the 'Open in split view' tab context menu item">
+          Open in split view
```

Comment:
I wonder if this should be a command too? Or does it only make sense on a tab which isn't the active tab?

Reply:
Yeah, could be. Does brave command always work for the active tab?

Reply:
I don't think there's any rules about it really - can use the focused tab, or the currently selected tab indexes

Reply:
Either way, not super important, feel free to ignore.